public/locales/en-US/grafana.json

```diff
@@ -13669,6 +13669,7 @@
         }
       }
     },
+    "query-validation-message": "One or more queries failed. The transformation will only use fields from successful queries.",
```

Comment:
Hidden queries can also trigger this. How about "One or more queries did not resolve with fields. This transformation can only use fields from queries have a visible, successful response." or something like that?